wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -459,4 +469,31 @@ const SingleRoute = (props: Props) => {
   );
 };
 
+const TransferDelayWarning = (props: {
+  token: string;
+  chain?: Chain;
+  duration: number;
+}) => {
+  const theme = useTheme();
+  const duration = formatDuration(props.duration);
+
+  return (
+    <div key="TransferDelayWarning">
+      <Divider flexItem sx={{ marginTop: '8px' }} />
+      <Stack direction="row" alignItems="center">
+        <WarningIcon htmlColor={theme.palette.warning.main} />
+        <Stack sx={{ padding: '16px' }}>
+          <Typography color={theme.palette.warning.main} fontSize={14}>
+            {`Your transfer ${
+              props.chain ? `to ${props.chain} ` : ''
+            }may be delayed due to rate limits set by ${
+              props.token
+            }. If your transfer is delayed, you will need to return after ${duration} to complete the transfer. Please consider this before proceeding.`}
```

Comment:
If this is an AutomaticTokenBridge transfer and it gets delayed by the governor, the transfer will be relayed once the VAA is emitted 24 hours later. The user doesn't need to return to complete it manually.

Reply:
I think this copy is ok. Might be too complicated/confusing to have different copy for manual and automatic.

Reply:
I think the text should say "after at most 24 hours", because enough room may free up in the rolling window before 24h (unless the tx is over the big tx limit but I guess we don't need to complicate this iteration)

Reply:
imo having a clear and consistent expectation (24 hours) even if its conservative is better than something that is not. As a user, I would be fine just checking back exactly the next day and seeing it done one time, vs thinking it *might* be done less than 24 hours and repeatedly checking in at random times throughout the day to be disappointed.